src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -3,6 +3,14 @@
 public class Ui {
     
     public void showLoadingError() {
```

Comment:
Maybe try to rename it like creatingNewFile to comply with the message show to the user

Reply:
Done